src/test/features/orders/form/models/disagreeReason.ts

```diff
@@ -0,0 +1,74 @@
+/* Allow chai assertions which don't end in a function call, e.g. expect(thing).to.be.undefined */
+/* tslint:disable:no-unused-expression */
+
+import { expect } from 'chai'
+import { Validator } from '@hmcts/class-validator'
+import { expectValidationError, generateString } from 'test/app/forms/models/validationUtils'
+import { ValidationErrors as DefaultValidationErrors } from 'forms/validation/validationErrors'
+import { ValidationConstraints } from 'forms/validation/validationConstraints'
+import { DisagreeReason } from 'orders/form/models/disagreeReason'
+
+describe('DisagreeReason', () => {
+
+  describe('constructor', () => {
+
+    it('should set the primitive fields to undefined', () => {
+      const reason = new DisagreeReason()
+      expect(reason.reason).to.be.undefined
+    })
+  })
+
+  describe('deserialize', () => {
+
+    it('should return an instance initialised with defaults for undefined', () => {
+      expect(new DisagreeReason().deserialize(undefined)).to.eql(new DisagreeReason())
+    })
+
+    it('should return an instance initialised with defaults for null', () => {
+      expect(new DisagreeReason().deserialize(null)).to.eql(new DisagreeReason())
+    })
+
+    it('should return an instance from given object', () => {
+      const description = 'I want a judge to review the order'
+      const result = new DisagreeReason().deserialize({
+        reason: description
+      })
+      expect(result.reason).to.be.equals(description)
+    })
+  })
+
+  describe('validation', () => {
+
+    const validator: Validator = new Validator()
+
+    it('should accept reason with empty string', () => {
+      const errors = validator.validateSync(new DisagreeReason(''))
+
+      expect(errors.length).to.equal(0)
```

Comment:
Arrays and collections can use `.to.be.empty` and `.to.have.lengthOf(number)`

Reply:
done